src/iterator.c

```diff
@@ -1716,6 +1724,8 @@ static void filesystem_iterator_update_ignored(filesystem_iterator *iter)
 GIT_INLINE(bool) filesystem_iterator_current_is_ignored(
 	filesystem_iterator *iter)
 {
+	if (iterator__skip_ignores(iter)) return false;
```

Comment:
I think libgit2 style is to put `return false` on a separate line? Same thing in a few other places.